pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -131,6 +142,18 @@ def __init__(self, filepath: Optional[str] = None, monitor: str = 'val_loss', ve
 
         self.kth_value, self.mode = mode_dict[mode]
 
+    @property
+    def best(self):
+        rank_zero_warn("attribute `best` has been renamed to `best_model_score` since v0.8.0"
+                       " and will be removed in v1.0.0", DeprecationWarning)
```

Comment:
```suggestion
        rank_zero_warn("Attribute `best` has been renamed to `best_model_score` since v0.8.0"
                       " and will be removed in v0.10.0", DeprecationWarning)
```